clib/exninfo.ml

```diff
@@ -57,12 +57,29 @@ let rec find_and_remove_assoc (i : int) = function
     if rem == ans then (r, l)
     else (r, (j, v) :: ans)
 
-let iraise e =
+type backtrace = Printexc.raw_backtrace
+let backtrace_to_string = Printexc.raw_backtrace_to_string
+
+let backtrace_info : backtrace t = make ()
```

Comment:
One question is whether we should use this or make `info` a record that may store the backtrace directly.